app/controllers/search_controller.rb

```diff
@@ -0,0 +1,5 @@
+class SearchController < ApplicationController
+  def search
```

Comment:
specify what kind of search it is
to understand what records info will be returned

Reply:
It seems I have corrected.